### PR TITLE
Select first item by default

### PR DESCRIPTION
**Pull request by GCorbel**
By default the selected item was not set with there is no prompt which can cause problems when there is no prompts.

### CODE REVIEW

File: addon/components/em-select.js [hunk 1]
```diff
@@ -56,18 +69,22 @@ export default Component.extend(InputComponentMixin, {
...
+      if (selectedIndex < 0) return;
+
       // check whether we show prompt the correct to show index is one less
       // when selecting prompt don't change anything
       if(this.get('prompt')){
         if(selectedIndex !== 0){
           selectedIndex--;
         } else {
-          this.set('model.' + this.get('property'), null);
+          model.set(this.get('property'), null);
```
Comment (spruce) [inline addon/components/em-select.js]: Nice cleanup 👍

File: addon/components/em-select.js [hunk 2]
```diff
@@ -32,8 +32,21 @@ export default Component.extend(InputComponentMixin, {
   autofocus: null,
   size: 0,
 
-  didReceiveAttrs() {
-    this._super(...arguments);
+  didRender() {
+    run.schedule('sync', this, () => {
```
Comment (spruce) [inline addon/components/em-select.js]: why do we need that?

Comment (GCorbel) [inline addon/components/em-select.js]: It's because it fail with older version of ember. It say we are changing a property in a didReceiveAttrs.

Comment (spruce) [inline addon/components/em-select.js]: ok. Thanks for the work. merging now.

**Pull request merged**